include/range/v3/view_interface.hpp

```diff
@@ -90,6 +90,26 @@ namespace ranges
             {
                 return Cardinality == 0;
             }
+            template<typename D = Derived,
+                CONCEPT_REQUIRES_(
+                    Cardinality < 0 && Cardinality != infinite &&
+                    !ForwardRange<D>() &&
+                    SizedRange<D>())>
+            RANGES_CXX14_CONSTEXPR bool empty()
```

Comment:
This qualifies for C++11 `constexpr`.

Reply:
Actually, not this one due to the lack of `const`.

Reply:
??

Reply:
The other `empty` overload you added can be `constexpr` in C++11, so you should replace `RANGES_CXX14_CONSTEXPR` for `constexpr`. Not this one, though.